services/meta/parameters.md

```diff
@@ -0,0 +1,149 @@
+# Parameters Service
+
+Parameters service allows configuration of a device.
+
+## Service name
+
+`parameters`
+
+## Interfaces
+
+| Type | Interface                 | Value type | Description                                                                                                  |
+|------|---------------------------|------------|--------------------------------------------------------------------------------------------------------------|
+| in   | cmd.sup_params.get_report | null       | Requests information about known parameters.                                                                 |
+| out  | evt.sup_params.report     | object     | Reports known parameters of the device, contains array of [`sup_parameter`](#definitions) objects.           |
+| in   | cmd.param.set             | object     | Sets parameter, see [`parameter_value`](#definitions).                                                       |
```

Comment:
so we cannot set multiple parameters at once?

Reply:
Good point, though I think you don't have a need for it. As modern UI is usually applying changes on the fly and does not behave like old-school forms. A loose idea @PaulinaDrozda: 

what if we made this bunch of interfaces:

cmd.param.set -> evt.param.report (for single param, and storage per param ID)
cmd.param.get -> evt.param.report (for single param, and storage per param ID)

optionally if adapter allows it:

cmd.params.get -> evt.params.report (get all parameters, and just one storage)

Just a loose idea - maybe we can recap once again...